### PR TITLE
Create form

### PR DESCRIPTION
**Pull request by eng-mohamed-eldeeb**
Create a footer form section. Add an h1 and a p paragraph. Add a form name e-mail inputs and a massage textarea. Add a submit button.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+                required
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                class="userInfo"
+                name="userMail"
+                placeholder="You'r email"
+              />
```
Comment (brytebee) [inline index.html]: - [ ] **Required:** This input box should be `required` like others. As it is at the moment a user can submit the form without providing their email address, thus, one essence for validation - data integrity is defeated. Kindly include the attribute here..

File: index.html [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="nav">
+      <a href="#" class="logo"><h1>My logo</h1></a>
+      <nav><i class="fas fa-bars menu_icon"></i></nav>
+    </header>
```
Comment (brytebee) [inline index.html]: - [ ] **Required:** I have observed that your design doesn't conform to that of the Figma template you've chosen. I have shown some of them below:

Your Design | Figma Template
:---: | :---:
image | image 
image | image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+        <div class="form-title">
+          <h3>Contact Me</h3>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form action="https://formspree.io/f/mrgrvdkl" method="post">
+          <ul class="ulclass">
```
Comment (brytebee) [inline index.html:248]: - [ ] **Required:** As much as possible your design should be similar to your chosen template as stated earlier.

Your Design | Figma Template 
:---: | :---:
image | image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, Team👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- good Github flow✔️
- descriptive commit messages✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,31 @@
-# protofolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HELLO MICROVERSE
```
Comment (andersonlebon) [inline README.md:3]: ## [OPTIONAL]

- [ ] to make your `README` more professional and to not confuse visitors I would suggest you give your project an appropriate title.

File: README.md [hunk 5]
```diff
@@ -1 +1,31 @@
...
+
+> AN HTML AND CSS FILE.
+
+![screenshot](./IMG1.png)
+
```
Comment (andersonlebon) [inline README.md:8]: ## [OPTIONAL]

- [ ] to make your `README` look more professional I suggest using a valid screenshot of the project. The current one is broken.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,286 @@
...
+            </li>
+            <li>
+              <textarea
+                class="userInfo user-massage"
+                name="user-name mass"
+                placeholder="Write your message here"
+                maxlength="500"
+                required
+              ></textarea>
+            </li>
```
Comment (andersonlebon) [inline index.html:276]: ## [OPTIONAL]

- [ ] Well done 👏  on styling this form, your design is looking good but I can see that we can improve it by putting a padding-top around the input `<textarea>`.  check here 👇 

image

**Review by andersonlebon — COMMENTED** [review]
## Hi @eng-mohamed-eldeeb 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 
- Your design is looking good ✔️  👍  💯 
## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**